### PR TITLE
Add an example for a quota tracked project

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Example resources that let you do the following:

1. set limits to a project
2. set a quota to a project
3. use an application template where frontend and database use parameterized resource limits

README to describe it all.

### CODE REVIEW

File: examples/project-quota/README.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+      capabilities: {}
+      privileged: false
+  dnsPolicy: ClusterFirst
+  restartPolicy: Always
+```
+
+Applying a Project Quota
+-----------------------------------------
+Administrators may want to restrict how much of the cluster resources a given project may consume across
+all of its pods.  To do this, an administrator applies a quota to a project.  A quota let's the
```
Comment (smarterclayton) [inline examples/project-quota/README.md]: `lets`

**Comment by smarterclayton** [comment]
Looks good in general.  One minor typo, merge whenever you fix.

**Comment by derekwaynecarr** [comment]
[merge]

**Pull request merged**